libraries/AP_BattMonitor/AP_BattMonitor_SMBus.cpp

```diff
@@ -2,6 +2,34 @@
 
 #define AP_BATTMONITOR_SMBUS_PEC_POLYNOME 0x07 // Polynome for CRC generation
 
+// read word from register
+// returns true if read was successful, false if failed
+bool AP_BattMonitor_SMBus::read_word(uint8_t reg, uint16_t& data) const
+{
+    // buffer to hold results (1 extra byte returned holding PEC)
+    const uint8_t read_size = 2 + (_pec_supported ? 1 : 0);
+    uint8_t buff[read_size];    // buffer to hold results
+
+    // read three bytes and place in last three bytes of buffer
```

Comment:
It was already wrong, but can you correct the comment? Just having *read register* might be the better option.